packages/playwright/src/worker/workerMain.ts

```diff
@@ -399,6 +399,8 @@ export class WorkerMain extends ProcessRunner {
         firstAfterHooksError = firstAfterHooksError ?? error;
       }
 
+      testInfo._tracing.didFinishTestFunctionAndHooks();
```

Comment:
Maybe add a comment why it is not called after 'afterAll' fixture / fixture teardowns below?